jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/KubernetesMockServerUtil.java

```diff
@@ -49,6 +54,15 @@ public static void prepareMockWebServerExpectationsForAggregatedDiscoveryEndpoin
       .always();
   }
 
+  public static void prepareMockWebServerExpectationsForOpenApiV3Endpoints(KubernetesMockServer server) throws IOException {
+    server.expect().get().withPath("/openapi/v3?timeout=32s")
+      .andReturn(200, IOUtils.toString(Objects.requireNonNull(KubernetesMockServerUtil.class.getResourceAsStream("/util/kubernetes-openapi-v3-schema.json")), StandardCharsets.UTF_8.toString()))
+      .always();
+    server.expect().get().withPath("/openapi/v3/api/v1?hash=64470CFAF8CA1AC72CDF17D98F7AB1B4FA6357371209C6FBEAA1B607D1B09E70C979B0BA231366442A884E6888CF86F0205FF562FCA388657C7250E472112154&timeout=32s")
+      .andReturn(200, IOUtils.toString(Objects.requireNonNull(KubernetesMockServerUtil.class.getResourceAsStream("/util/kubernetes-openapi-v3-api-v1-schema-pod.json")), StandardCharsets.UTF_8.toString()))
+      .always();
+  }
```

Comment:
Is there any helm CLI flag that might prevent the checking/validation altogether so that we don't have to mock these jsons?

Reply:
Helm does have a `--disable-openapi-validation` flag to disable the OpenAPI validation. We are already providing this during building HelmConfig since we implemented Helm Install functionality.

However, helm CLI doesn't seem to be respecting this flag while processing hooks. See [pkg/action/hooks.go](https://github.com/helm/helm/blob/3a3e3846ca9c929a6966583b461181e70f19bc13/pkg/action/hooks.go#L59) , second argument to `Build` is for OpenAPI Schema Validation, it's hardcoded to `true`